Cargo.toml

```diff
@@ -40,6 +40,11 @@ tree-sitter = { version = "0.20", git = "https://github.com/tree-sitter/tree-sit
 nucleo = "0.2.0"
 
 [workspace.package]
+version = "23.10.0"
```

Comment:
Yes, it throws an error if the version is not semver. But it looks like having a patch version is a thing in CalVer, so it can be compatible with both.